src/Analyzers/CSharp/CodeFixes/UseCollectionExpression/CSharpUseCollectionExpressionForBuilderCodeFixProvider.cs

```diff
@@ -82,6 +83,15 @@ internal partial class CSharpUseCollectionExpressionForBuilderCodeFixProvider()
         foreach (var (statement, _) in analysisResult.Matches)
             subEditor.RemoveNode(statement);
 
+        //Add the leading trivia from the declaration of the builder to the invocation where we convert the builder to a collection.
+        var dummyObjectCreationDeclaration = dummyObjectCreation.Ancestors().Take(3);
+
+        if (dummyObjectCreationDeclaration.Last() is VariableDeclarationSyntax variableDeclaration)
```

Comment:
```suggestion
        if (dummyObjectCreationDeclaration.LastOrDefault() is VariableDeclarationSyntax variableDeclaration)
```
`Last()` crashes if there are no elements in the sequence. The desired behavior here was to use `LastOrDefault()`

Reply:
I added LastOrDefault for safety, but I can't really think of cases where this will be empty?